docs/src/pages/docs/comparison.md

```diff
@@ -12,40 +12,47 @@ Feature/Capability Key:
+| Supported Query Keys                         | JSON                                   | JSON                       | GraphQL Query                         | GraphQL Query                         |
+| Query Key Change Detection                   | Deep Compare (Stable Serialization)    | Referential Equality (===) | Deep Compare (Unstable Serialization) | Deep Compare (Unstable Serialization) |
+| Query Data Memoization Level                 | Query + Structural Sharing             | Query                      | Query + Entity + Structural Sharing   | Query + Entity + Structural Sharing   |
+| Bundle Size                                  | [![][bp-react-query]][bpl-react-query] | [![][bp-swr]][bpl-swr]     | [![][bp-apollo]][bpl-apollo]          | [![][bp-react-relay]][bpl-react-relay]|
+| Queries                                      | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Caching                                      | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Devtools                                     | ✅                                     | 🟡                         | ✅                                    | ✅                                    |
+| Polling/Intervals                            | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Parallel Queries                             | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Dependent Queries                            | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Paginated Queries                            | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Infinite Queries                             | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Lagged / "Lazy" Queries<sup>1</sup>          | ✅                                     | 🛑                         | 🛑                                    | ✅                                    |
+| Initial Data                                 | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Scroll Recovery                              | ✅                                     | ✅                         | ✅                                    | 🛑                                    |
```

Comment:
What does scroll recovery mean in the case of data fetching libraries? Isn’t this a routing concern?

Reply:
I think Apollo does not check here green either

Reply:
Based on this documentation Relay checks here green too since it loads the old cache till something newer is loaded. E.g because of the Lagged / "Lazy" Queries<sup>1</sup> part